src/components/search/search.tsx

```diff
@@ -26,11 +23,27 @@ export const Search = () => {
 
   const dispatch = useAppDispatch();
 
-  const handleClick = () => {
+  const handleSearch = (searchText: string) => {
     navigate(`/search/${searchText}`);
     dispatch(fetchMoviesAction({ searchText, searchBy: 'title' }));
   };
 
+  return <SearchView onSearch={handleSearch} searchQuery={searchQuery} />;
+};
+
+export interface SearchViewProps {
+  onSearch: (searchText: string) => void;
+  searchQuery?: string;
+}
+export const SearchView: FC<SearchViewProps> = ({ onSearch, searchQuery }) => {
```

Comment:
Not a good idea to place 2 components in one file. Each file should have only single exported component, let's put it outside in separate file